frontend/src/components/Footer.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import '../../src/styles/style.css';
```

Comment:
Hi Pei, the footer style doesn't seem to be connected. Sorry I think my last comment might have been confusing.

Please could you remove the `syle.css` import line here and add
`@import url('./components/footer.css');` to the `style.css `file under Components section?

That way we don't need to import any css files into component files. We only have to import them to the `style.css` file, which is the only style file imported to `App.jsx`